index.html

```diff
@@ -0,0 +1,166 @@
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+  <!--Google fonts -> Lato, cdn fonts -> cocogoose-->
+
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+
+  <!-- page stylesheets -->
+
+  <link rel="stylesheet" href="stylesheets/mobile.css">
+  <link rel="stylesheet" href="stylesheets/desktop.css">
+
+  <title>Vision Straight Discovery</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="social-icons">
+      <ul>
+        <li><a href="https://github.com/carlylechia"><i class="fa fa-github" aria-hidden="true"></i>
+        </a></li>
+        <li><a href="https://www.linkedin.com/in/chia-carlyle/"><i class="fa fa-linkedin" aria-hidden="true"></i>
+        </a></li>
+        <li><a href="https://twitter.com/CarlyleChia"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">French</a></li>
+        <li><a href="#">logout</a></li>
+
+      </ul>
+    </nav>
+    <nav class="top-navbar">
+      <div class="logo">
+        <a href="index.html"><img src="images/VSB-logo.jpeg" alt="VSB logo"></a>
+      </div>
+      <div class="desktop-nav">
+        <ul>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#" class="bordered" >VSB Annual Festival</a></li>
+        </ul>
+      </div>
+      <div class="menu-btn">
+        <a href="#"><img id="menu" src="images/menu-icon.png" alt="menu-btn"></a>
+      </div>
+    </nav>
+
+    <nav class="mobile-nav">
+      <ul id="nav">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#">VSB Annual Festival</a></li>
+      </ul>
+    </nav>
+
+    <section class="landing">
+      <div class="landing-header">
+        <h3>"Vision Straight Records (VSR) Presents..."</h3>
+        <h1>Vision Straight Discovery Showdown 2022</h1>
+      </div>
+      <div class="landing-body">
+        <p>The Vision Straight Discovery (VSD) edition #1 2022 is a first of many shows to come, which aims at providing talented individuals a platform to showcase their musical talents, for a chance to become part of the next generation of superstars, and to be signed into the Vision Straight Records Brand, with loads of added advantages.</p>
+      </div>
+      <div class="landing-description">
+        <h2>2022.05.15(SUN) ~ 22(SUN)</h2>
+        <p>@ VS Concert Hall, Bamenda, Cameroon.</p>
+      </div>
+    </section>
+  </header>
```

Comment:
-[ ] [REQUIRED] In order to have a better user experience, kindly adjust the circular background image as it is currently obstructing the text.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![1](https://user-images.githubusercontent.com/58692590/153318772-75733a6b-5d5d-499e-9913-0cca5525a2ab.png) | ![11](https://user-images.githubusercontent.com/58692590/153318792-0522e40c-6866-43d0-b82e-2c6d9a549416.png)